packages/thirdweb/src/wallets/in-app/native/helpers/storage/state.ts

```diff
@@ -6,7 +6,3 @@ let cognitoUserAtom: CognitoUser | undefined;
 export function getCognitoUser() {
   return cognitoUserAtom;
```

Comment:
we use this stil? just never setting it?

Reply:
called in `logoutUser()`

Reply:
and it doesn't need to be, removing